### ISSUE TITLE
Last Hit Date incorrect timezone

### ISSUE THREAD

**Issue by darren131** [nystudio107/retour#79]
Hi,

My timezone in Craft CMS settings is set to UTC+12 (NZST) - Pacific/Auckland but the times in Retour are off by 12 hours...

My current time is 3:09pm the last hit was a few moments ago:
https://www.dropbox.com/s/pk3pfsh6rt346rz/Screenshot%202017-07-26%2015.10.08.png?dl=0

**Comment by khalwat** [comment]
Yeah it stores the redirects in UTC format, and doesn't make any attempt to localize them in the frontend. It probably should, I'll make this a feature request.
